lib/core/src/Cardano/Wallet/Primitive/AddressDerivation.hs

```diff
@@ -148,7 +151,7 @@ import qualified Data.Text.Encoding as T
 --
 -- We do not manipulate purpose, cointype and change paths directly, so they are
 -- left out of the sum type.
-data Depth = RootK | AccountK | AddressK
+data Depth = RootK | PurposeK | CoinTypeK | AccountK | RoleK | AddressK
```

Comment:
It seems the documentation doesn't apply anymore.

Reply:
Indeed!